### DISCUSSION TITLE
[Bug]: TextareaOnChangeData should be able to import from "@fluentui/react-components"

### DISCUSSION THREAD

**Discussion by XinyangWang1024** [microsoft/fluentui#26971] [General] [answered]
I am using "@fluentui/react-components": "9.10.1", but I can not import "TextareaOnChangeData" from "@fluentui/react-components".

![image](https://user-images.githubusercontent.com/37166032/221390898-d09a30cf-cf0d-4ace-a8cc-35c227bb0408.png)

Is there another way to import this type? or is this a small issue to be fixed? Thanks.

**Comment by layershifter** [comment, marked as answer]
It's available in latest versions: https://github.com/microsoft/fluentui/blob/a1e3381f27cb4c1cb1fb45c1d699395920f5afa7/packages/react-components/react-components/src/index.ts#L713

Was fixed in #26455.
